### PR TITLE
SRVCOM-2100 install serverless operator using cli

### PR DESCRIPTION
**Pull request by gabriel-rh**
Version(s):
4.8+

Issue:
https://issues.redhat.com/browse/SRVCOM-2100

Link to docs preview:
https://52908--docspreview.netlify.app/openshift-enterprise/latest/serverless/install/install-serverless-operator.html#serverless-install-cli_install-serverless-operator

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by skonto** [comment]
LGTM

File: modules/serverless-install-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  name: serverless-operator
+  namespace: openshift-serverless
+spec:
+  channel: <channel name> <1>
+  name: serverless-operator <2>
+  source: redhat-operators <3>
+  sourceNamespace: openshift-marketplace <4>
+----
+<1> The channel name of the Operator. The `stable` channel enables installation of the most recent stable version of the {ServerlessOperatorName}.
+<2> Name of the Operator to subscribe to.
```
Comment (cardil) [inline modules/serverless-install-cli.adoc:55]: This is always `serverless-operator`.

**Review by cardil — CHANGES_REQUESTED** [review]
Some minor requests.

**Comment by gabriel-rh** [comment]
> Some minor requests.

@cardil I have made the changes suggested and updated the preview here:

https://52908--docspreview.netlify.app/openshift-enterprise/latest/serverless/install/install-serverless-operator.html#serverless-install-cli_install-serverless-operator

**Review by jc-berger — APPROVED** [review]
/lgtm

**Review by cardil — APPROVED** [review]
/lgtm

from QE

**Pull request merged**